### PR TITLE
Send W3C capabilities in the new session request.

### PR DESCRIPTION
**Pull request by juangj**
If the Selenium standalone server only finds legacy capabilities in the request,
then it will assume that the client doesn't speak W3C, and will return a legacy
response. Sending a request that includes the W3C keys induces it to send a W3C
response.

- [X] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#step-6-sign-the-cla)

### CODE REVIEW

**Comment by juangj** [comment]
This probably also applies to Grid, but I'm really not that familiar with Grid so I'm not positive.

Other language bindings do fancier things like filtering non-W3C capability names out of the W3C capabilities blob ([here it is in Python](https://github.com/SeleniumHQ/selenium/blob/master/py/selenium/webdriver/remote/webdriver.py#L43)). Not sure how necessary you feel that is.

Also kinda undecided how to treat required capabilities, but those should go away at some point anyway, so maybe you were already planning an API change to eliminate them.

File: javascript/node/selenium-webdriver/firefox/index.js [hunk 1]
```diff
@@ -380,7 +380,7 @@ function normalizeProxyConfiguration(config) {
     }
   } else if ('pac' === config.proxyType) {
     if (config.proxyAutoconfigUrl && !config.pacUrl) {
-      config.pacUrl = config.proxyAutoconfigUrl;
+      config.proxyAutoconfigUrl = config.proxyAutoconfigUrl;
```
Comment (jleyba) [inline javascript/node/selenium-webdriver/firefox/index.js]: Actually, it looks like the entire `normalizeProxyConfiguration` function isn't needed anymore

**Comment by juangj** [comment]
`npm test` passed (or, well, it didn't fail any more than it already did on my machine). And this seemed to work for many tests, but it doesn't work for tests that set non-W3C capabilities.

I do have some code (probably living in the wrong file right now) that will filter non-W3C capability names like some of the other language bindings do. But if you're planning to completely ditch legacy protocol support anyway, then maybe there's no need for any of this -- once all the capability names are changed, you could just switch completely to W3C capabilities instead of sending this hybrid.

(I'm going to backport some version of this to our internal 3.6.0 branch in order to continue to support Chrome users, though.)

**Comment by juangj** [comment]
I added that commit that drops invalid capabilities, if you're curious.

The `normalizeProxyConfiguration` thing should probably just go in a separate commit.

**Pull request merged**